AnnoyancesFilter/sections/popups.txt

```diff
@@ -129,6 +129,7 @@ levi.com##.opt-in-email-signup-drawer ~ div.modal-backdrop
 brighteon.com#%#document.cookie="popupSubscription=1; path=/;";
 webmd.com##.nl-hov-mobile
 newizv.ru##.popup
+speakenglishwell.ru###overlay-sidebar-wrap
```

Comment:
Duplicate it to the Social filter.
Also remove a social widget (leftover)
<details><summary>Screenshot:</summary>

![image](https://user-images.githubusercontent.com/33488229/91275329-ae0ede80-e788-11ea-8b8a-fdc73a23a78d.png)

</details><br/>